### PR TITLE
docs: move host selection strategies to separate page

### PR DESCRIPTION
**Pull request by jasonli-improving**
### Summary

Moved host selection strategies out of UsingTheReadWriteSplittingPlugin to separate page

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/using-the-jdbc-driver/ReaderSelectionStrategies.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Reader Selection Strategies
+By default, the read/write plugin randomly selects a reader instance the first time that `setReadOnly(true)` is called. To balance connections to reader instances more evenly, different selection strategies can be used. The following table describes the currently available selection strategies and any relevant configuration parameters for each strategy.
```
Comment (sergiyvamz) [inline docs/using-the-jdbc-driver/ReaderSelectionStrategies.md:2]: Can we remove references about RW splitting plugin? I think reader (or in general, node) selection strategies are used driver wide, by many plugins.

File: docs/using-the-jdbc-driver/ReaderSelectionStrategies.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+To indicate which selection strategy to use, the `readerHostSelectorStrategy` configuration parameter can be set to one of the selection strategies in the table below. The following is an example of enabling the least connections strategy:
+
```
Comment (sergiyvamz) [inline docs/using-the-jdbc-driver/ReaderSelectionStrategies.md:5]: This should be left on RW splitting plugin page

File: docs/using-the-jdbc-driver/ReaderSelectionStrategies.md (hunk 1 above)
Comment (jasonli-improving) [reply docs/using-the-jdbc-driver/ReaderSelectionStrategies.md]: Removed the reference to the RW plugin

File: docs/using-the-jdbc-driver/ReaderSelectionStrategies.md (hunk 2 above)
Comment (jasonli-improving) [reply docs/using-the-jdbc-driver/ReaderSelectionStrategies.md]: Moved this back

**Review by sergiyvamz — APPROVED** [review]

**Pull request merged**